_topic_map.yml

```diff
@@ -784,67 +784,79 @@ Name: Operators
-  File: olm-what-operators-are
-- Name: Common terms
-  File: olm-common-terms
-- Name: Packaging formats
-  File: olm-packaging-format
-- Name: Understanding Operator Lifecycle Manager (OLM)
-  Dir: understanding_olm
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-  Topics:
-  - Name: Concepts
-    File: olm-understanding-olm
-  - Name: Architecture
-    File: olm-arch
-  - Name: Workflow
-    File: olm-workflow
-  - Name: Dependency resolution
-    File: olm-understanding-dependency-resolution
-  - Name: OperatorGroups
-    File: olm-understanding-operatorgroups
-  - Name: Metrics
-    File: olm-understanding-metrics
-- Name: Understanding OperatorHub
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-  File: olm-understanding-operatorhub
-- Name: Adding Operators to a cluster
-  File: olm-adding-operators-to-cluster
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
-- Name: Configuring proxy support
-  File: olm-configuring-proxy-support
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-- Name: Deleting Operators from a cluster
-  File: olm-deleting-operators-from-cluster
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
-- Name: Creating applications from installed Operators
-  File: olm-creating-apps-from-installed-operators
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
-- Name: Viewing Operator status
-  File: olm-status
-  Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
-- Name: Creating policy for Operator installations and upgrades
-  File: olm-creating-policy
-  Distros: openshift-origin,openshift-enterprise,openshift-webscale
-- Name: Managing admission webhooks in OLM
-  File: olm-webhooks
-  Distros: openshift-origin,openshift-enterprise,openshift-webscale
-- Name: Managing custom catalogs
-  File: olm-managing-custom-catalogs
-  Distros: openshift-origin,openshift-enterprise,openshift-webscale
-- Name: Using OLM on restricted networks
-  File: olm-restricted-networks
-  Distros: openshift-origin,openshift-enterprise,openshift-webscale
-- Name: CRDs
-  Dir: crds
+  Dir: understanding
+  Topics:
+  - Name: What are Operators?
+    File: olm-what-operators-are
+  - Name: Common terms
+    File: olm-common-terms
+  - Name: Packaging formats
+    File: olm-packaging-format
+  - Name: Operator Lifecycle Manager (OLM)
+    Dir: olm
+    Distros: openshift-enterprise,openshift-webscale,openshift-origin
+    Topics:
+    - Name: Concepts
+      File: olm-understanding-olm
+    - Name: Architecture
+      File: olm-arch
+    - Name: Workflow
+      File: olm-workflow
+    - Name: Dependency resolution
+      File: olm-understanding-dependency-resolution
+    - Name: OperatorGroups
+      File: olm-understanding-operatorgroups
+    - Name: Metrics
+      File: olm-understanding-metrics
+  - Name: OperatorHub
```

Comment:
![scrollbar](https://user-images.githubusercontent.com/16197393/94176491-2b9c3c00-fe66-11ea-91c6-a1eb144183fc.png)

Reply:
Same with the Configuring a custom PKI topic

Reply:
Sadly happening already in published docs: https://docs.openshift.com/container-platform/4.5/operators/olm-understanding-operatorhub.html

:(

Started after a recent CSS update; might be related to when an assembly ends with a callout. Need to investigate more, but outside of the scope of this PR.

Reply:
Was able to work around this by moving the "Additional resources" list to the bottom of the TOC... :smirk: